File: onlineexamination/src/main/java/com/cg/onlineexamination/student/dto/Examdto.java
package com.cg.onlineexamination.student.dto;

import java.time.LocalDate;
import java.util.ArrayList;

import com.cg.onlineexamination.entity.TestPaper;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@NoArgsConstructor
@AllArgsConstructor

public class Examdto {

	private int examId;
	private int score;
	private LocalDate dateOfExam;

	private int testId;
	private int questionNo;
	private String studentAnswer;
	
	
	public int getExamId() {
		return examId;
	}
	public void setExamId(int examId) {
		this.examId = examId;
	}
	public int getScore() {
		return score;
	}
	public void setScore(int score) {
		this.score = score;
	}
	public LocalDate getDateOfExam() {
		return dateOfExam;
	}
	public void setDateOfExam(LocalDate dateOfExam) {
		this.dateOfExam = dateOfExam;
	}
	public int getTestId() {
		return testId;
	}
	public void setTestId(int testId) {
		this.testId = testId;
	}
	public int getQuestionNo() {
		return questionNo;
	}
	public void setQuestionNo(int questionNo) {
		this.questionNo = questionNo;
	}
	public String getStudentAnswer() {
		return studentAnswer;
	}
	public void setStudentAnswer(String studentAnswer) {
		this.studentAnswer = studentAnswer;
	}
	
	
	

}

File: src/app/exam-dto.ts
export class ExamDto {

    examId:number;
	score:number;
	dateOfExam:number;
    testId:number;
    questionNo:number;
    studentAnswer:string;



    constructor(
    examId:number,
	score:number,
	dateOfExam:number,
    testId:number,
    questionNo:number,
    studentAnswer:string,

    ){
        this.examId = examId;
        this.score = score;
        this.dateOfExam = dateOfExam;
        this.testId=testId;
        this.questionNo=questionNo;
        this.studentAnswer = studentAnswer;
    }
}
